core/dbt/contracts/graph/unparsed.py

```diff
@@ -411,11 +411,11 @@ class ExposureOwner(JsonSchemaMixin, Replaceable):
 
 
 @dataclass
-class UnparsedExposure(JsonSchemaMixin, Replaceable):
+class UnparsedExposure(HasYamlMetadata, Replaceable):
     name: str
+    description: str = ''
```

Comment:
Whoops — `description` needs to go below, between `owner` and `maturity`, because it has a default argument, and for whatever reason args with defaults _must_ follow args without defaults. This is on me for suggestion otherwise

Reply:
@joellabes Ugh, sorry, I just realized that `get_rendered` [here](https://github.com/fishtown-analytics/dbt/pull/2920/files#diff-a25a77362db78f4ddc0675bcbd4775e158631076c9918ce5b64cfcd9f6db6bceR628) is operating on a `description` field from `ParsedExposure`, not `UnparsedExposure`. And that those are different! Silly me.

Could you do the same switcheroo here, replacing it with `description: str = ''`:
https://github.com/fishtown-analytics/dbt/blob/5ba5271da99bc1ef7fbad2f7d0b45634087300fa/core/dbt/contracts/graph/parsed.py#L659

Reply:
Yep! While also leaving it in its changed state for UnparsedExposure?

Reply:
Yes, I think we should keep those two consistent, even if there isn't direct inheritance between them

Reply:
So close :) one failing unit test, `test_basic_parsed_exposure `. I think that's just a matter of adding a default `description: ''` to these:
https://github.com/fishtown-analytics/dbt/blob/5ba5271da99bc1ef7fbad2f7d0b45634087300fa/test/unit/test_contracts_graph_parsed.py#L2006-L2059